docs/content/direct/get-started.md

```diff
@@ -86,6 +96,8 @@ After that cleanup, you may want to `set -e` so that failures do not
 go unnoticed (the various cleanup commands may legitimately "fail" if
 there is nothing to clean up).
 
+
+
```

Comment:
We have gotten complaints from Markdown linters about multiple consecutive blank lines. They render the same as a single blank line. I recommend not annoying linters this way.